src/main/java/se/sundsvall/supportmanagement/api/ErrandCommunicationsAttachmentResource.java

```diff
@@ -42,14 +42,15 @@ class ErrandCommunicationsAttachmentResource {
 	}
 
 	@Operation(summary = "Get a streamed communication attachment.", description = "Fetches the communication attachment that matches the provided id in a streamed manner")
-	@GetMapping(path = "/{attachmentID}/streamed", produces = {ALL_VALUE, APPLICATION_PROBLEM_JSON_VALUE})
+	@GetMapping(path = "/{attachmentId}/streamed", produces = {
+		ALL_VALUE, APPLICATION_PROBLEM_JSON_VALUE
+	})
 	@ApiResponse(responseCode = "200", description = "Successful Operation", useReturnTypeSchema = true)
 	@ApiResponse(responseCode = "404", description = "Not Found", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 	void getMessageAttachmentStreamed(
 		@Parameter(name = "namespace", description = "Namespace", example = "my.namespace") @Pattern(regexp = NAMESPACE_REGEXP, message = NAMESPACE_VALIDATION_MESSAGE) @PathVariable final String namespace,
 		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId,
-		@Parameter(name = "attachmentID", description = "Message attachment id", example = "b82bd8ac-1507-4d9a-958d-369261eecc15") @ValidUuid @PathVariable final String attachmentID, final HttpServletResponse response) {
-		service.getMessageAttachmentStreamed(namespace, municipalityId, attachmentID, response);
+		@Parameter(name = "attachmentId", description = "Message attachment ID", example = "b82bd8ac-1507-4d9a-958d-369261eecc15") @ValidUuid @PathVariable final String attachmentId, final HttpServletResponse response) {
+		service.getMessageAttachmentStreamed(namespace, municipalityId, attachmentId, response);
```

Comment:
Should this still be here? Is there a point to have one with errand and one without?

`/{municipalityId}/{namespace}/errands/{errandId}/communication/{communicationID}/attachments/{attachmentID}/streamed`

and

 `/{municipalityId}/{namespace}/communication/attachments/{attachmentId}/streamed`

Reply:
Removced `/{municipalityId}/{namespace}/communication/attachments/{attachmentId}/streamed` after discussion with frontend.

They are not using any of theses resources now, but wanted to keep one for future requirements.